src/Microsoft.DotNet.CodeFormatting/IFormattingEngine.cs

```diff
@@ -16,7 +16,7 @@ public interface IFormattingEngine
         ImmutableArray<string> FileNames { get; set; }
         ImmutableArray<IRuleMetadata> AllRules { get; }
         ImmutableArray<DiagnosticDescriptor> AllSupportedDiagnostics { get; }
-        bool AllowTables { get; set; }
+        bool AllowTables { get;  set; }
```

Comment:
Only change to this file is now introduction of extraneous white space. Just revert the file.